### DISCUSSION TITLE
Cyberduck 9.1.1 (42639) Unable to Connect to Google Drive

### DISCUSSION THREAD

**Discussion by willgreene999** [iterate-ch/cyberduck#16733] [Q&A] [answered]
Just downloaded on my Windows 11 pc. Licensed Purchased. Trying to connect to my Google Drive, after opening web page in Chrome, selecting my Google account, Cyberduck gives failed to connect error. 

The error is: Connection Failed - Network is unreachable. The connection attempt was rejected. The server may be down, or the network may not be properly configured.

![image](https://github.com/user-attachments/assets/81dab1e4-63d5-495e-976e-70e7339b7d7b)

Any help would be appreciated.

**Comment by willgreene999** [comment]
Log file attached
[cyberduck.log](https://github.com/user-attachments/files/18276230/cyberduck.log)

**Reply by willgreene999** [reply, marked as answer]
I installed version 9.1.0 and that allows me to connect now.

**Comment by ylangisc** [comment]
This is caused by #16723.
